tests/providers/opensearch/hooks/test_opensearch.py

```diff
@@ -0,0 +1,37 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from airflow.providers.opensearch.hooks.opensearch import OpenSearchHook
+
+MOCK_SEARCH_RETURN = {"status": "test"}
+
+
+class TestOpenSearchHook:
+    def test_hook_search(self, mock_hook):
```

Comment:
Does this actually work if you only run the tests in this file?   I wold have expected to need to import the mock_hook fixture

Reply:
and the fixture doesn't have autouse=True....   I wouldn't expect this to work as written

Reply:
Yes this works. See https://docs.pytest.org/en/6.2.x/fixture.html#scope-sharing-fixtures-across-classes-modules-packages-or-session

Reply:
autouse=True would mean that tests do not have to explicitly request the fixture by having it defined as a parameter for the test function. I am just being explicit by requesting the fixture but I could switch it to autouse.

Reply:
If it works, it works.   I must have my scopes mixed up.  My mistake.

Reply:
Here is a output in breeze running just a single test. 
```
root@a8147a85fe4a:/opt/airflow/tests/providers/opensearch/hooks# pytest test_opensearch.py -k "test_hook_search"
========================================= test session starts =========================================
platform linux -- Python 3.10.13, pytest-7.4.2, pluggy-1.3.0 -- /usr/local/bin/python
cachedir: .pytest_cache
rootdir: /opt/airflow
configfile: pyproject.toml
plugins: httpx-0.21.3, mock-3.11.1, xdist-3.3.1, capture-warnings-0.0.4, instafail-0.5.0, anyio-4.0.0, rerunfailures-12.0, timeouts-1.2.1, asyncio-0.21.1, cov-4.1.0, time-machine-2.13.0, requests-mock-1.11.0
asyncio: mode=strict
setup timeout: 0.0s, execution timeout: 0.0s, teardown timeout: 0.0s
collected 2 items / 1 deselected / 1 selected                                                         

test_opensearch.py::TestOpenSearchHook::test_hook_search PASSED                                 [100%]

=================================== 1 passed, 1 deselected in 0.43s ===================================
```